lib/active_model/serializer.rb

```diff
@@ -17,32 +19,10 @@ class Serializer
     include Associations
     include Attributes
     include Caching
+    include Links
+    include Type
```

Comment:
this pattern will likely bite us if we continue it indefinitely, but I like how it's helping us understand the code better right now